app/workers/zomato_data_grabbing_worker.rb

```diff
@@ -0,0 +1,16 @@
+class ZomatoDataGrabbingWorker
+  include Sidekiq::Worker
+  include Faraday
```

Comment:
I don't think you should include Faraday here. 

You're not using it as far as I can see, and even if you did -  you can create an instance of Faraday client and use it to make the http calls.